pkg/users/account-create-dialog.js

```diff
@@ -78,6 +80,23 @@ function AccountCreateBody({ state, errors, change }) {
                     value={state.home_dir} />
             </FormGroup>
 
+            <FormGroup label={_("Shell")}
+                       fieldId="accounts-create-user-shell">
+                <Select variant={SelectVariant.single}
+                        toggleId="accounts-create-user-shell"
+                        onToggle={statusIsExpanded => change("isShellSelectExpanded", statusIsExpanded)}
+                        onSelect={(event, selection) => { change("shell", selection); change("isShellSelectExpanded", false) }}
+                        selections={shell}
+                        isOpen={isShellSelectExpanded}
+                        aria-labelledby="vm-state-select"
+                        menuAppendTo="parent"
+                        isCreatable
+                        isCreateOptionOnTop
+                        createText={_("Use")}>
```

Comment:
When/where would that be visible? It's not part of the screenshots.

Reply:
Ah nice catch, that is a leftover from previous refactoring, not applicable anymore, will remove it.